core/src/main/java/io/questdb/griffin/engine/functions/math/NegIntFunctionFactory.java

```diff
@@ -71,6 +71,11 @@ public int getInt(Record rec) {
             return value != Numbers.INT_NULL ? -value : Numbers.INT_NULL;
         }
 
+        @Override
```

Comment:
@bluestreak01 this causes the single test failure in `CompiledFilterRegressionTest.testColumnArithmeticsNullComparison` in a single case:
`result mismatch for query: select * from x where i64+-i32`

Expected returns rows where `i64==null` but actual returns rows where both `i64==null AND i32==null`.

When I delete this function, this test passes